draft-dekater-scion-dataplane.md

```diff
@@ -284,7 +284,7 @@ Segments cannot be combined arbitrarily. To construct a valid forwarding path, t
 
 Besides enabling the enforcement of path policies, the above rules also protect the economic interest of ASes, as they prevent building "valley paths". A valley path contains ASes that do not profit economically from traffic on this route. The name comes from the fact that such paths go "down" (following parent-child links) before going "up" (following child-parent links).
 
-{{figure-1}} below shows the different allowed segment combinations.
+{{figure-1}} below shows the possible segment combinations.
```

Comment:
"possible" doesn't really convey the effect of other choices. After all one can totall craft incorrect packets. How about "valid"? That's what the router does, after all: it "validates" the path.

Reply:
Changed to Valid

Reply:
Changed to Valid